src/MICore/CommandFactories/lldb.cs

```diff
@@ -160,5 +160,12 @@ public override Task Catch(string name, bool onlyOnce = false, ResultClass resul
         {
             throw new NotImplementedException("lldb catch command");
         }
+
+        public async override Task<string> VarAssign(string variableName, string expression, int threadId, uint frameLevel)
+        {
+            Results results = await VarCreate(expression, threadId, frameLevel, 0, ResultClass.done);
```

Comment:
The EVALFLAGS get used in the clrdbg code path. Since this is the lldb command factory, you know that the eval flags will never be used, so it seems okay to just pass zero.